dpnp/backend/custom_kernels_statistics.cpp

```diff
@@ -198,13 +200,25 @@ void custom_mean_c(void* array1_in, void* result1, const size_t* shape, size_t n
         return;
     }
 
-    _DataType* sum = reinterpret_cast<_DataType*>(dpnp_memory_alloc_c(1 * sizeof(_DataType)));
+    if constexpr (std::is_same<_DataType, double>::value || std::is_same<_DataType, float>::value)
+    {
+        _ResultType* array = reinterpret_cast<_DataType*>(array1_in);
+        auto dataset = mkl_stats::make_dataset<mkl_stats::layout::row_major>(1, size, array);
 
-    custom_sum_c<_DataType>(array1_in, sum, size);
+        cl::sycl::event event = mkl_stats::mean(DPNP_QUEUE, dataset, result);
```

Comment:
Could you please add a comment with documentation pointer to this functionality from MKL?

Reply:
something like 
`// http://asdada`